demo/src/Components/utils.ts

```diff
@@ -10,3 +11,31 @@ export function updateBodyClassName(theme: Theme) {
     bodyEl.classList.toggle('g-root_theme_light', theme === 'light');
     bodyEl.classList.toggle('g-root_theme_dark', theme === 'dark');
 }
+
+const isBrowser = () => typeof document !== 'undefined';
+
+function getMobileView() {
+    if (!isBrowser()) {
+        return false;
+    }
+
+    return document.body.clientWidth < 768;
+}
+
+export function useMobile() {
```

Comment:
consider using

```
import {useMobile} from '@gravity-ui/uikit';
```

Reply:
@martyanovandrey not working with useMobile, storybook must  have effect on change viewport size. this hook does't. 
I tested it doesn't work